### ISSUE TITLE
CSS instead of "functions"?

### ISSUE THREAD

**Issue by conradbielitz** [portato-io/portatoapp#119]
For example in `RouteSummary.tsx` there is this:

```
  const containerHeight = window.innerHeight * 0.7;
  return (
    <PageLayout>
      <ProgressBar progress={PROGRESS} />
      <div
        style={{
          marginTop: '5vh',
          height: containerHeight + 'px',
          overflowY: 'auto',
        }}
      >

```

This should be achievable with a CSS rule like `div .summary {height:100vh}`. The benefits would be:
- Less code
- Re-usable for other areas, like the Sender>Summary.tsx

As always: Maybe this was done for a specific reason, in which case; ignore/close this issue ^^

**Comment by mAkeddar** [comment]
I totally agree with the re-usable part ! For the moment, we're using a mix of inline styling and CSS for 1 main reason:
- Some of our stylings depend on variables, and I still didn't figure out how to import those variables into CSS files 
Definitely a task for after the 28th :)
